.eslintrc.js

```diff
@@ -25,76 +25,82 @@ module.exports = {
     'plugin:react/recommended',
   ],
 
-  plugins: [
-    'babel',
-    'react',
-    'json',
-    'import',
-  ],
+  plugins: ['babel', 'react', 'json', 'import', 'prettier'],
 
   globals: {
-    '$': 'readonly',
+    $: 'readonly',
     document: 'readonly',
     QUnit: 'readonly',
     window: 'readonly',
   },
 
   rules: {
+    'prettier/prettier': 'error',
+    'space-before-function-paren': [
+      'error',
+      { anonymous: 'always', named: 'never' },
+    ],
+    indent: 'off',
```

Comment:
I tried modifying indent rules to match prettier, but the recommendation from prettier is just to disable formatting rules entirely and so I did ( and came here with this snippet to justify me taking the easy road )